content/yaml-publishing/steam.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Steam
+description: How to deploy an app to Steam using codemagic.yaml
+weight: 15
+
+---
+
+Codemagic enables you to automatically publish your Windows/MacOS/Linux desktop app to Steam.
+
+{{<notebox>}}
+This guide only applies to workflows configured with **codmegic.yaml**.
+{{</notebox>}}
+
+{{<notebox>}}
+A Steam Partner account is required to create and configure an app to appear on Steam.
+{{</notebox>}}
+
+### Upload to Steam
+
+steamcmd is a tool used to upload builds to Steam, and can be installed as part of the build workflow using:
```

Comment:
Here and elsewhere: is it SteamCMD or `steamcmd`?

Reply:
Updated to SteamCmd everywhere, unless used as a direct code example.  This matches the convention used in Steam documentation.